Option to save checkpoint for best evaluation result

emailweixu opened the pull request.

emailweixu committed:
Option to save checkpoint for best evaluation result

Haichao-Zhang reviewed (dismissed):
Great feature! Thanks!

emailweixu committed:
Add sac_pendulum.gin back

Some other tests use it.
Also improved BestEvalChecker to be more general

emailweixu committed:
Truly add back sac_pendulum.gin

emailweixu committed:
Raise FileNotFoundError if the checkpoint cannot be found

This is needed for global_step='best' so that the latest one
can be loaded instead.

emailweixu committed:
Fix unittest
